### PR TITLE
fix: new comment

### PR DESCRIPTION
**Pull request by muslim-kareem**
Changelist: 

- Added a comment

### CODE REVIEW

File: backend/src/main/java/com/example/backend/message/MessageRestController.java [hunk 1]
```diff
@@ -20,6 +20,7 @@ public Message getMessageByAuthorId(@PathVariable String id){
         return service.getMessageByAuthorId(id);
     }
 
+    //muslim was here
```
Comment (AdrianSchulze) [inline backend/src/main/java/com/example/backend/message/MessageRestController.java]: This should say: adrian was here

**Review by AdrianSchulze — CHANGES_REQUESTED** [review]

File: backend/src/main/java/com/example/backend/message/MessageRestController.java (hunk 1 above)
Comment (AdrianSchulze) [reply backend/src/main/java/com/example/backend/message/MessageRestController.java]: hello

Comment (muslim-kareem) [reply backend/src/main/java/com/example/backend/message/MessageRestController.java]: i fixed it mr Adrian

**Review by AdrianSchulze — APPROVED** [review]
lgtm

**Pull request merged**